src/main/java/org/jenkinsci/remoting/engine/JnlpAgentEndpointResolver.java

```diff
@@ -107,22 +108,15 @@ public class JnlpAgentEndpointResolver extends JnlpEndpointResolver {
     private static String PROTOCOL_NAMES_TO_TRY =
             System.getProperty(JnlpAgentEndpointResolver.class.getName() + ".protocolNamesToTry");
 
-    public JnlpAgentEndpointResolver(String... jenkinsUrls) {
-        this.jenkinsUrls = new ArrayList<>(Arrays.asList(jenkinsUrls));
-    }
-
-    public JnlpAgentEndpointResolver(@NonNull List<String> jenkinsUrls) {
-        this(jenkinsUrls, null, null, null, null, false);
-    }
```

Comment:
These overloads appear unused, so I did not bother retaining compatibility for them.